.circleci/config.yml

```diff
@@ -29,6 +29,19 @@ jobs:
           root: .
           paths: site
 
+  build_docs_pdf:
+    working_directory: ~/bids-specification/pdf_build_src
+    docker:
+      - image: danteev/texlive:TL2017
+    steps:
+      - checkout:
+          path: ~/bids-specification
+      - run: 
```

Comment:
I think you may be creating yourself some problems with the paths you are specifying in `working_directory` and `checkout`.

I would leave the `working_directory` line alone (i.e., remove line 33 here) ... then we will have the default working directory of `~/project` (see https://discuss.circleci.com/t/working-directory-is-no-longer-required-defaults-to-project/14363/2)

Then I would also remove line 38, corresponding to the `path` parameter of the `checkout` setting ... that will mean that the `bids-specification` will be cloned (checked out) to the default directory, which is the working directory, which will be `~/project`

Then you should take a look at your python scripts and how they handle paths. All the while assuming that you are working from a directory `~/project/`